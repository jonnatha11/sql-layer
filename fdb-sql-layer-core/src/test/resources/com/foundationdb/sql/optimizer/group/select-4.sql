SELECT child.name FROM child WHERE pid IN (SELECT parent.id FROM parent WHERE state = 'MA')
