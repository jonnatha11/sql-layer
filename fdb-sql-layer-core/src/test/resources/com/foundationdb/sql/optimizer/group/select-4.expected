SELECT _G_1.child$name AS name FROM test._akiban_parent AS _G_1 WHERE (_G_1.parent$state = 'MA') AND (TRUE AND TRUE)